scripts/run-benchmarks.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
```

Comment:
maybe add a comment that say that if you modify this script, you might need to modify also
run-pro-benchmarks.sh